src/components/AvailableBalanceUI/index.tsx

```diff
@@ -1,16 +1,20 @@
 import clsx from 'clsx';
+import InterlayButtonBase from 'components/UI/InterlayButtonBase';
+import React from 'react';
 
 import { POLKADOT, KUSAMA } from 'utils/constants/relay-chain-names';
 
-interface Props {
+interface Props extends React.HTMLAttributes<unknown> {
```

Comment:
What's this giving us – is it to let us pass through `onClick`?

Reply:
Yes, allow the component to inherit all the native attrs such as className, onClick, onBlur and so on.